members.html

```diff
@@ -85,6 +85,7 @@ <h1 class="c-heading">Member organizations</h1>
         <a href="https://janus.conf.meetecho.com" class="c-members-link" target="_blank" rel="external"><img src="members/logo-janus.png" alt="Janus" class="c-members-logo"></a>
         <a href="https://github.com/sipcapture/homer" class="c-members-link" target="_blank" rel="external"><img src="members/logo-homer.png" alt="Homer" class="c-members-logo"></a>
         <a href="https://www.cavellgroup.com/" class="c-members-link" target="_blank" rel="external"><img src="members/logo-cavell.png" alt="Cavell Group" class="c-members-logo"></a>
+        <a href="https://www.deanone.nl/" class="c-members-link" target="_blank" rel="external"><img src="members/logo-deanone.png" alt="Dean One" class="c-members-logo"></a>
```

Comment:
You're linking to a `.png` here but the source file is `.svg`, if you change this I'll merge it! :+1:

Reply:
Committed.